nemo/collections/asr/models/rnnt_bpe_models.py

```diff
@@ -62,6 +62,8 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
                 "container that supports RNN-T loss."
             )
 
+        cfg = model_utils.setup_model_config(cfg)
```

Comment:
what is the difference between `setup_model_config` and `convert_model_config`?

Reply:
(guess will learn that soon ;))

Reply:
Im bad at naming things. But basically setup converts {DictConfig, ModelPTConfig dataclass} to DictConfig (which earlier was hardcoded into ModelPT and problematic for cases like *BPE where we need to inject stuff before actually callin super()).

convert will take a config and map it to Hydra 1.x config if necessary.

Reply:
so how about simply go with `to_dict_config()`?

Reply:
ehehe, it be too close to `to_config_dict()` that we already have in Serialization... :]

Reply:
> to_config_dict()
Yep thats why I avoided that name, safer to have "common" things inside model_utils.py